components/x-teaser/src/Image.jsx

```diff
@@ -17,20 +17,33 @@ const aspectRatio = ({ width, height }) => {
 	return null;
 };
 
-const NormalImage = ({ src }) => (
-	<img className="o-teaser__image" src={src} alt="" />
+
+const NormalImage = ({ src, srcSet }) => (
+	srcSet
+	? <img className="o-teaser__image" src={src} srcSet={srcSet} alt="" />
+	: <img className="o-teaser__image" src={src} alt="" />
 );
 
-const LazyImage = ({ src, lazyLoad }) => {
+const LazyImage = ({ src, srcSet, lazyLoad }) => {
 	const lazyClassName = typeof lazyLoad === 'string' ? lazyLoad : '';
-	return <img className={`o-teaser__image ${lazyClassName}`} data-src={src} alt="" />;
+	return srcSet 
+	? <img className={`o-teaser__image ${lazyClassName}`} data-src={src} srcSet={srcSet} data-srcSet={srcSet} alt="" />
+	: <img className={`o-teaser__image ${lazyClassName}`} data-src={src} alt="" />;
 };
 
-export default ({ relativeUrl, url, image, imageSize, imageLazyLoad, ...props }) => {
+export default ({ relativeUrl, url, image, imageSize, imageLazyLoad, imageHighDensity, ...props }) => {
 	const displayUrl = relativeUrl || url;
 	const useImageService = !(image.url.startsWith('data:') || image.url.startsWith('blob:'));
+	const optionsHighDensity = {
+		quality: 'highest',
```

Comment:
to me there's absolutely no perceptible difference between `highest` and `medium` quality on 2x screens. i don't think this warrants the extra markup output, download size, and code complexity.

## `medium`
<img width="1194" alt="screenshot of how to spend it with medium quality images" src="https://user-images.githubusercontent.com/631757/92572701-af5a0400-f27c-11ea-8409-dee42299235b.png">


## `highest`
<img width="1198" alt="screenshot of how to spend it with highest quality images" src="https://user-images.githubusercontent.com/631757/92572612-92bdcc00-f27c-11ea-95ca-497592eec012.png">